src/google/protobuf/compiler/java/java_message_builder.cc

```diff
@@ -473,26 +473,32 @@ GenerateCommonBuilderMethods(io::Printer* printer) {
     "public Builder setField(\n"
     "    com.google.protobuf.Descriptors.FieldDescriptor field,\n"
     "    Object value) {\n"
+    "  //@@protoc_insertion_point(field_modifier_scope_after:$full_name$.setField)\n"
```

Comment:
Please don't add insertion points here. These methods are not meant to be customized.

Reply:
Indeed we actually didn't need to customize those methods. The insertion points are no longer present in latest pull request